sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -76,6 +76,10 @@ public class ServiceBusProcessor : IAsyncDisposable
         /// <summary>
         /// Gets the <see cref="ReceiveMode"/> used to specify how messages are received. Defaults to PeekLock mode.
         /// </summary>
+        /// <value>
+        /// The receive mode can be set using <see cref="ServiceBusProcessorOptions.ReceiveMode"/>
```

Comment:
```suggestion
        /// The receive mode is specified using <see cref="ServiceBusProcessorOptions.ReceiveMode"/>
```

Reply:
good catch.   I missed that when I was updating the verbiage!